Project.toml

```
@@ -31,7 +31,7 @@ LogExpFunctions = "0.3"
 NaNMath = "1"
 Preferences = "1"
 SpecialFunctions = "1, 2"
-StaticArrays = "1.5"
+StaticArrays = "< 1.7"
```

This will re-enable StaticArrays support for all versions < 1.5. I think it would be safer to only allow
```suggestion
StaticArrays = "1.5, 1.6"
```